<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="main">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="m_signupButton">
         <property name="text">
          <string>SIGN UP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_loginButton_2">
         <property name="text">
          <string>LOG IN</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="login">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="m_emailLabel">
           <property name="text">
            <string>Email:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_emailLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="m_passwordLabel">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_passwordLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="m_loginButton">
         <property name="text">
          <string>LOGIN</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="registration">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="m_firstNameLabel">
             <property name="text">
              <string>First name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_firstNameLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="m_lastNameLabel">
             <property name="text">
              <string>Last name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_lastNameLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="m_emailLabelRegistration">
             <property name="text">
              <string>Email:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_emailRegistrationLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="m_passwordLabelRegistration">
             <property name="text">
              <string>Password:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_passwordLineEditRegistration"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="m_repassword">
             <property name="text">
              <string>Password confirm:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_repasswordLineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QGroupBox" name="m_sexGroupBox">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="title">
            <string>Sex:</string>
           </property>
           <widget class="QRadioButton" name="m_maleRadioButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>69</width>
              <height>18</height>
             </rect>
            </property>
            <property name="text">
             <string>Male</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="m_femaleRadioButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>100</width>
              <height>18</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Female</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="m_isAdminCheckBox">
           <property name="text">
            <string>Admin</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="m_daylabel">
           <property name="text">
            <string>Day:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_monthlabel">
           <property name="text">
            <string>Month:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_yearlabel">
           <property name="text">
            <string>Year:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QComboBox" name="m_daycombobox"/>
         </item>
         <item>
          <widget class="QComboBox" name="m_monthcomboBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="m_yearcomboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="m_registerButton">
         <property name="text">
          <string>SIGN UP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="profile">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="m_first_name_label">
           <property name="text">
            <string>First name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_last_name_label">
           <property name="text">
            <string>Last name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_email_label">
           <property name="text">
            <string>Email:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_sexLabel">
           <property name="text">
            <string>Sex:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_birthDate_label">
           <property name="text">
            <string>BirthDate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_admin_label">
           <property name="text">
            <string>Admin:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_active_label">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 127);</string>
           </property>
           <property name="text">
            <string>Active:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="m_tasks_button">
         <property name="text">
          <string>Task list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_editProfilebutton">
         <property name="text">
          <string>Edit profile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_logout_button">
         <property name="text">
          <string>Log out</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="editprofile">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="m_first_name_label_2">
           <property name="text">
            <string>First name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_editfirstnamelineedit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="m_last_name_label_2">
           <property name="text">
            <string>Last name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_editlastnamelineedit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="m_email_label_2">
           <property name="text">
            <string>Email:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_editemaillineedit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="m_passwordlabel">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_editpasswordlineedit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="m_passwordconfirmlabel">
           <property name="text">
            <string>Confirm password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_editpasswordconfirmlineedit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="m_submitbutton">
         <property name="text">
          <string>SUBMIT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
